.github/scripts/check_packages.py

```diff
@@ -0,0 +1,15 @@
+import setuptools
+import pprint
+
+where = "./src"
+package_list = set(setuptools.find_packages(where))
+namespace_packages = set(setuptools.find_namespace_packages(where))
+diff1 = package_list.difference(namespace_packages)
+if len(diff1) > 0:
+    pprint.pprint(diff1)
+diff2 = namespace_packages.difference(package_list)
+if len(diff2) > 0:
```

Comment:
Let’s just implement this as part of pytest instead of adding another workflow

Reply:
The added benefit is that pytest will generate the debug diff details on failure automatically

Reply:
done